homeassistant/components/technove/diagnostics.py

```diff
@@ -0,0 +1,27 @@
+"""Diagnostics support for TechnoVE."""
+
+from __future__ import annotations
+
+from dataclasses import asdict
+from typing import Any
+
+from homeassistant.components.diagnostics import async_redact_data
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_HOST, CONF_MAC
+from homeassistant.core import HomeAssistant
+
+from .const import DOMAIN
+from .coordinator import TechnoVEDataUpdateCoordinator
+
+TO_REDACT = {CONF_HOST, CONF_MAC, "unique_id", "mac_address", "entry_id"}
+
+
+async def async_get_config_entry_diagnostics(
+    hass: HomeAssistant, entry: ConfigEntry
+) -> dict[str, Any]:
+    """Return diagnostics for a config entry."""
+    coordinator: TechnoVEDataUpdateCoordinator = hass.data[DOMAIN][entry.entry_id]
+    return async_redact_data(
+        {"entry": entry.as_dict(), "technove-data": asdict(coordinator.data.info)},
```

Comment:
I am wondering, what useful information do you get from the config entry? The only thing in the data is already redacted

Reply:
You're right, that's not necessary at the moment. I removed it.